src/include/daos/agent.h

```diff
@@ -26,15 +26,30 @@
 
 #include <daos/drpc.h>
 
+int dc_agent_init(void);
+void dc_agent_fini(void);
```

Comment:
These need doxygen comments

Reply:
None of the other ones have comments but I added them anyway.